templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-main.yaml

```diff
@@ -312,12 +331,12 @@ Resources:
         - 'CAPABILITY_AUTO_EXPAND'
       StackInstancesGroup:
         - Regions:
-            - !Ref CloudTrailRegions
+            - !Ref CloudTrailRegion
           DeploymentTargets:
             Accounts:
               - !Ref "AWS::AccountId"
       TemplateURL: !Sub
-        - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml'
+        - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}${QSVersion}/templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml'
```

Comment:
Fix the `AllowedPattern: "^[0-9a-zA-Z-/]*$"` for QSS3KeyPrefix such that it always ends with `/`

Reply:
fixed